src/views/view_vehicle_control.cpp

```diff
@@ -331,13 +331,17 @@ namespace big
 			ImGui::EndTooltip();
 		}
 
+		ImGui::BeginDisabled(g.window.vehicle_control.no_max_summon);
 		ImGui::SliderFloat("Max summon distance", &g.window.vehicle_control.max_summon_range, 10.f, 250.f);
 		if (ImGui::IsItemHovered())
 		{
 			ImGui::BeginTooltip();
 			ImGui::Text("At what range the vehicle will drive towards the summoned location as oposed to being teleported");
 			ImGui::EndTooltip();
 		}
+		ImGui::EndDisabled();
+
+		ImGui::Checkbox("No Max Summon Distance", &g.window.vehicle_control.no_max_summon);
```

Comment:
Can be made into a bool command.

Reply:
This is great. I love using the vehicle summon for RP purposes (like a getaway vehicle being driven to me to get out of a hairy situation), but the 250M limit sometimes breaks my immersion.

Hope this gets into the main build soon!

Reply:
> Can be made into a bool command.

Which part of it? the whole thing(summoning)

Reply:
The line I highlighted, it can be changed into a `bool_command`.

Reply:
```cpp
// register command
bool_command g_ignore_max_summon_distance("vehcontrolignoremaxsummondist", "Ignore Max Summon Distance", "Enabling this will allow summoning your last vehicle from any distance", g.window.vehicle_control.no_max_summon);

// Show checkbox in UI
components::command_checkbox<"vehcontrolignoremaxsummondist">();
```

It's not that hard, here is another example from in the codebase:
https://github.com/YimMenu/YimMenu/blob/be7bf8ffa6f969c1850bd44b5c1b8d4c5ba7bb66/src/views/view_vehicle_control.cpp#L318-L330

Reply:
Okay, thanks. I have been a bit busy but I will get to it.